web/template/investigation_on_hold.gohtml

```diff
@@ -52,7 +52,7 @@
                             <div class="govuk-radios__item">
                                 <input class="govuk-radios__input" id="f-reason-2" name="reason" type="radio" value="LA Investigation" {{ if eq "LA Investigation" .Reason }}checked{{ end }}>
                                 <label class="govuk-label govuk-radios__label" for="f-reason-2">
-                                    LA Investigation
+                                    Local Authority Investigation
```

Comment:
Small content tweaks from speaking to Claire & Katie

Reply:
Although this won't match what is displayed for `.HoldPeriod.Reason` on the take investigation off hold page